### PR TITLE
 Add CloudFormation template 

### PR DESCRIPTION
**Pull request by gene1wood**
Also  update README to capture changes in how the service is deployed and where

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,4 +66,5 @@ Logs for the webserver are located in
 `/usr/local/openresty/nginx/logs/error.log`
 `/usr/local/openresty/nginx/logs/access.log`
 
-These logs are not rotated and will eventually fill the 8GB disk.
+These logs are rotated with a manuall provisioned `/etc/logrotate.d/openresty`
```
Comment (floatingatoll) [inline README.md]: nit: manually

**Review by floatingatoll — COMMENTED** [review]
This seems valid enough. I've added a couple other folks to get eyes on this, because this is definitely far outside of 'usual approach' for my team, and I don't have a lot of personal experience with CF templates — but it's also live in prod, so I don't think I should block anyways.

**Review by bkochendorfer — APPROVED** [review]
CF template looks good to me. It might be helpful to throw a note in the README about how you use the CF template. For example, are we copy pasting it into the console, applying via the CLI, etc. That way we know how to reproduce and edit going forward

File: README.md (hunk 1 above)
Comment (gene1wood) [reply README.md]: Aha, thanks for catching that, I've fixed it in 79d6f55903d36fc9f0ec8ad092d276a140b58f4b

**Comment by gene1wood** [comment]
@bkochendorfer I've added a section to the README that talks about how to deploy a CloudFormation template in 79d6f55903d36fc9f0ec8ad092d276a140b58f4b

**Review by duallain — APPROVED** [review]

**Pull request merged**